### ISSUE TITLE
Cryptic error messages returned in OK messages

### ISSUE THREAD

**Issue by boreq** [hoytech/strfry#83]
Strfry tends to return cryptic messages in `OK` messages justifying rejecting events. For example when I a `p` tag doesn't actually contain a hex public key the following message will be returned: 

>invalid: uneven size input to from_hex

Now this can be deciphered by investigating the event but it would be much better if the error messages were friendlier e.g.:

>invalid: event contains a p tag with an invalid public key

**Comment by hoytech** [comment]
I completely agree with this -- the error messages are pretty terrible. I just need to put the work in to improving this. Right now they are whatever exception messages the various libraries I use throw.

I'll see if I can make some improvements to this for the 1.0 release.

**Comment by hoytech** [comment]
Next release it will error with: `invalid: unexpected size for fixed-size tag: p`. If it's exactly 64 characters but it fails on hex decode it will say `invalid: unexpected character in from_hex: 122`.

Not perfect, but getting better :)
